curriculum/challenges/espanol/02-javascript-algorithms-and-data-structures/functional-programming/use-the-some-method-to-check-that-any-elements-in-an-array-meet-a-criteria.md

```diff
@@ -1,50 +1,51 @@
 ---
 id: 587d7dab367417b2b2512b6f
-title: Use the some Method to Check that Any Elements in an Array Meet a Criteria
+title: Usa el método "some" para comprobar si algún elemento en un arreglo cumple un criterio
```

Comment:
@nhcarrigan Here is another example of the double quotes.

Reply:
I need to test what happens if we use backticks in a title. 🤔